crates/astria-sequencer-relayer/src/data_availability.rs

```diff
@@ -280,59 +332,47 @@ impl CelestiaClient {
             .wrap_err("failed signing sequencer namespace data")?
             .to_bytes()
             .wrap_err("failed converting signed namespace data to bytes")?;
-        let resp = self
-            .submit_namespaced_data(&DEFAULT_NAMESPACE.to_string(), &bytes)
+        // TODO: Could this also be thrown into the submission above?
```

Comment:
I believe so based on Celestia documentation, again solving #122 likely requires this all be a single large batch, and it should be better economically to have a single batch (and it should be a single batch per celestia blocktime)

Reply:
Gotcha, in that instance w may need updates to format pushed for everything at the same height, unless we have a solution to nonce issues.

Reply:
@noot Ohhh, that's perfect. That will make the fix to #122 so nice.